banjax_integration_test.go

```diff
@@ -15,6 +15,20 @@ func TestMain(m *testing.M) {
 	os.Exit(exit_code)
 }
 
+func TestGlobalPerSiteDecisionListsMask(t *testing.T) {
+	prefix := "/auth_request?path="
+	httpTester(t, []TestResource{
+		// we should not treat CIDR as normal IP, will be skipped in map
+		{"GET", prefix + "/global_mask_noban", 200, ClientIP("192.168.1.0/24"), nil},
```

Comment:
hi @jeremy5189, I think this header should change to use a valid IP in the range of `192.168.1.0/24`, instead of sending the subnet itself.